app/views/pages/cookie_consent.html.erb

```diff
@@ -0,0 +1,82 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Cookies</h1>
+
+    <p class="govuk-body">Request for personal information puts small files (known as ‘cookies’) onto your device to collect information about how you browse the site.</p>
```

Comment:
just noticed you've got some smart apostrophes in here, I think maybe they aren't standard for govuk design so I normally replace them with the normal straight one